waypoints:
  - id: 1
    position:
      x: 6.131731033325195
      y: 3.9944944381713867
    euler_angle:
      z: 1.57
    functions:
      - function: variable_waypoint_radius
        waypoint_radius: 0.5
    robot_wait: false 
  - id: 2
    position:
      x: 4.68256950378418
      y: 6.624748229980469
    euler_angle:
      z: 0.785
    robot_wait: true 
  - id: 3
    position:
      x: 2.188396453857422
      y: 7.045740604400635
    euler_angle:
      z: 0.0
    robot_wait: false 
